src/utils/configBuilders.ts

```diff
@@ -44,12 +45,16 @@ export function buildRollupConfigData({
   rollupContracts,
   validators,
   batchPoster,
+  parentChainId,
 }: {
   rollupConfig: RollupConfig;
   rollupContracts: RollupContracts;
   validators: Wallet[];
   batchPoster: Wallet;
+  parentChainId: number;
 }): RollupConfigData {
+  const parentChainRpcUrl = getRpcUrl(parentChainId);
+
   return {
     'chain': {
       'info-json': [
```

Comment:
https://github.com/OffchainLabs/orbit-deployment-ui/blob/450aed9f88443320f3cc6c548d743159830107cd/src/utils/configBuilders.ts#L63
also need to change this

Reply:
Fixed in https://github.com/OffchainLabs/orbit-deployment-ui/pull/52/commits/e9744e50e9fe667c400becbab6a4664fbb22ab13